### ISSUE TITLE
Session missing in custom analytics events

### ISSUE THREAD

**Issue by floric** [aws/aws-amplify#256]
Hey,
this is a question or bug:

Currently custom events with Analytics.record() don't include the current session. That might be the reason why they don't show up in Mobile Analytics and Pinpoint (at least for me?!).
Is this by design?
Based on https://docs.aws.amazon.com/mobileanalytics/latest/ug/generating-analytics-events.html I assume it should be correct and easy to add the session to the event.

If you like, I will create a PR!

**Comment by mlabieniec** [comment]
@floric The events you send (including session start/stop etc) will show up in the Amazon Pinpoint console under Campaign Analytics -> Events. You can get there also from the mobile hub console and clicking on Analytics in the top right of your project. Are you not seeing your events there?

**Comment by floric** [comment]
Ah, thanks. That's where I expected them. But before switching from MA to Pinpoint I used events named like "APP/LOAD_PAGE". They were recorded but never showing up in this list. Thats why I thought it might be because of the session information. Today I renamed them and upgraded to 0.2.4 (thanks BTW!) and now they are showing up. Feel free to close it.
Maybe it was because of the slash in the name? Or they were sent but maybe not processed in the AWS backend? Not sure, what has happened. They are now showing up perfectly. :)
